inc/crt/lnetdb.bi

```diff
@@ -0,0 +1,160 @@
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+	ai_addr as sockaddr ptr
+	ai_addrlen as socklen_t
+	ai_canonname as zstring ptr
+	ai_family as long
+	ai_flags as long
+	ai_next as addrinfo ptr
+	ai_protocol as long
+	ai_socktype as long
+	h_addr_list as zstring ptr ptr
+	h_addrtype as long
+	h_aliases as zstring ptr ptr
+	h_length as long
+	h_name as zstring ptr
+	n_addrtype as long
+	n_aliases as zstring ptr ptr
+	n_name as zstring ptr
+	n_net as uint32_t
+	p_aliases as zstring ptr ptr
+	p_name as zstring ptr
+	p_proto as long
+	s_aliases as zstring ptr ptr
+	s_name as zstring ptr
+	s_port as long
+	s_proto as zstring ptr
+#define EAI_ADDRFAMILY -9
+#define EAI_AGAIN -3
+#define EAI_BADFLAGS -1
+#define EAI_FAIL -4
+#define EAI_FAMILY -6
+#define EAI_MEMORY -10
+#define EAI_NODATA -5
+#define EAI_NONAME -2
+#define EAI_OVERFLOW -12
+#define EAI_SERVICE -8
+#define EAI_SOCKTYPE -7
+#define EAI_SYSTEM -11
+#define HOST_NOT_FOUND 1
+#define NETDB_INTERNAL -1
+#define NETDB_SUCCESS 0
+#define NI_DGRAM 16
+#define NI_MAXHOST 1025
+#define NI_MAXSERV 32
+#define NI_NAMEREQD 8
+#define NI_NOFQDN 4
+#define NI_NUMERICHOST 1
+#define NI_NUMERICSERV 2
+#define NO_ADDRESS 4
+#define NO_DATA 4
+#define NO_RECOVERY 3
+#define SCOPE_DELIMITER asc( "%" )
+#define TRY_AGAIN 2
+#define _PATH_HEQUIV "/etc/hosts.equiv"
+#define _PATH_HOSTS "/etc/hosts"
+#define _PATH_NETWORKS "/etc/networks"
+#define _PATH_NSSWITCH_CONF "/etc/nsswitch.conf"
+#define _PATH_PROTOCOLS "/etc/protocols"
+#define _PATH_SERVICES "/etc/services"
+#define __crt_linux_netdb_bi__
+#define h_errno *__h_errno_location()
+#endif
+#ifndef __crt_linux_netdb_bi__
+#include once "crt/netinet/in.bi"
+#include once "crt/stdint.bi"
+''
+''
+''
+''
+''
+''         be included in other distributions without authorization.
+'' NOTICE: This file is part of the FreeBASIC Compiler package and can't
+'' bits\netdb -- header translated with help of SWIG FB wrapper
```

Comment:
What's going on here? Where have these headers come here? Copy-pasted from other headers, with or without hand edits? Are any of these new headers actually machine-translated, or these NOTICEs all copy-pasted?

Reply:
@rversteegen Good to hear from you. These are your headers, that you pointed me to. I apologize for not discussing this with you. Some cut and pastes were made in response to real world testing that I mentioned in #290.

Reply:
Hi; sorry for disappearing so long.

Wow, you're right, these totally butchered changes to the include files are my fault! After a lot of head scratching, I see that  `inc/crt/fnetdb.bi` and `inc/crt/lnetdb.bi` are not real headers. They're just garbage versions of `netdb.bi` that I was I using to diff, which I checked into to git accidentally in commit https://github.com/rversteegen/fbc/commit/8082ee67320527ddeba8b443de89874255f8ba6f, because I didn't give my changes even a cursory glance. These header changes were definitely not meant to be merged; they need a lot of work. The other changes to the three `netdb.bi` files may be alright, but I wouldn't count on it. There are also many other bad include file changes.

I suggest that you remove all include file changes from this PR, and we start over on them.

Now that Jeff has started some work on FreeBSD include files I think it's preferable to continue the way he's going, which is to have separate versions of each header for each OS, rather than a single file with sections gated with `#ifdef`s (as the fbbindings 'crt' branch does, using fbfrog). This avoids needing to download headers for *all* OSes and run a huge fbfrog job to update anything.

Reply:
Agreed. I’ll remove the headers from this PR.

Reply:
All headers and changes to headers have been removed.